### ISSUE TITLE
Regarding independent testing 

### ISSUE THREAD

**Issue by shrutiOx** [GenomeNet/Genome-NAS#1]
Hi Team,

Thank you for this great work! I just needed to understand more about the testing process. Generally in a genomic deep-learning based application, we keep an independent test dataset and the model obtained from training is applied on this test-dataset and we report the performance. In this case, I did not understand how are test results are achieved in the "(iv) Evaluation phase:" of the cycle? Did you again train your final model on the training dataset (original train split) and then tested on the test split or did you do a K-FOLD CV like approach on the test dataset itself which would mean if am not wrong training on the testing dataset itself in each fold and getting results? Could you kindly clarify this please. Especially I want to understand, why are we training the model again  in the "Evaluation phase", when we have it from "(iii) Selection phase:" ?

And to further clarify this point mentioned in your paper (pg 8:table 1) - " TRAINING WAS DONE WITH SUBSAMPLED EPOCHS, SO VALUES MAY NOT BE COMPARABLE WITH OTHER VALUES IN THE LITERATURE"
So  if a model is trained and tested on the same dataset (DeepSEA), will the test-set results thus obtained, can  directly be compared against your results that you have provided in table-1 , assuming those indicates indepdendent test-set (DeepSea test set) verification results. Please let me know!

Thanks a lot for your kind consideration to this question in advance!
